README.md

```diff
@@ -126,6 +139,28 @@ Write code to do the following:
 Write your code here:
 ```ruby
 # code here
+genie = {
+  name: "Genie"
+}
+
+jafar {
+  name: "Jafar"
+}
+
+lamp = {
+  wishes_remaining: 3,
+  genie: [genie[:name]]
+}
+
+lamp[:wishes_remaining] = 1
+
+jafarLamp = {
+  wishes_remaining: 3,
+  genie: [jafar[:name]]
+}
+
+lamp = nil
+
 ```
```

Comment:
Nice use of nested data! I'm bad about this, too, but it's worth noting that Rubyists tend to prefer snake_case.